docs/book/SUMMARY.md

```diff
@@ -31,7 +33,10 @@
     * [`.expect().verify()`](API/Expectation-API.md#verify)
   * Classes
     * [`Server`](API/Server.md)
-* [Recipes/Examples](https://github.com/mockyeah/mockyeah/tree/master/examples)
-* [CLI](Service-Snapshot-CLI.md)
+* [Snapshots](Snapshots/Overview.md)
+  * [Snapshots Overview](Snapshots/Overview.md#snapshots-overview)
+  * [Ad Hoc Snapshots](Snapshots/Overview.md#ad-hoc-snapshots)
+  * [CLI](Snapshots/CLI.md)
```

Comment:
What if the CLI isn't just for snapshots in the future - does it still make sense under this section for now?